core/db/queries.py

```diff
@@ -533,6 +533,33 @@ def risk_register_query(base_query: Query) -> Query:
     return extended_query
 
 
+def project_finance_change_query(base_query: Query) -> Query:
+    """
+    Extend base query to select specified columns for ProjectFinanceChange.
+
+    Joins to ProjectFinanceChange model table (not included in base query joins)
+
+    :param base_query: SQLAlchemy Query of core tables with filters applied.
+    :return: updated query.
+    """
+    extended_query = (
+        base_query.join(
+            ents.ProjectFinanceChange, ents.ProjectFinanceChange.programme_junction_id == ents.ProgrammeJunction.id
+        )
+        .with_entities(
+            ents.Submission.submission_id,
+            ents.Programme.programme_id,
+            ents.ProjectFinanceChange.data_blob,
+            ents.Project.project_name,
```

Comment:
I don't think this field makes sense now (Project.project_name). The query should pull the PFC project names from the jsonb and differentiate them

Reply:
Makes sense